app/helpers/pulfalight_helper.rb

```diff
@@ -132,6 +132,15 @@ def regular_compact_breadcrumbs(document)
     )
   end
 
+  # Provides the search algorithm value for the UI component
+  def search_algorithm_value
```

Comment:
Can we add default to allowed_search_algorithms and check that somehow?

Reply:
Done.

Reply:
Done